build.go

```diff
@@ -212,6 +212,40 @@ func (b *Build) GetConsoleOutput() string {
 	return content
 }
 
+// GetProgressiveText gets progressive text with the param in previous response header.
```

Comment:
What is `previous response header`? The comment is confused, please make it clearer.